rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -200,38 +200,33 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCOCOFileSourcePartialSingleShard
                                                             unsigned shard_id,
                                                             unsigned shard_count,
                                                             bool is_output,
+                                                            std::vector<float>& area_factor,
+                                                            std::vector<float>& aspect_ratio,
+                                                            unsigned num_attempts,
                                                             bool shuffle = false,
                                                             bool loop = false,
-                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MAX_SIZE,
-                                                            unsigned max_width = 0, unsigned max_height = 0,
-                                                            RocalFloatParam area_factor = NULL, RocalFloatParam aspect_ratio = NULL,
-                                                            RocalFloatParam y_drift_factor = NULL, RocalFloatParam x_drift_factor = NULL);
+                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_USER_GIVEN_SIZE_RESTRICTED,
+                                                            unsigned max_width = 0, unsigned max_height = 0 );
 
-/// Creates JPEG image reader and decoder. It allocates the resources and objects required to read and decode COCO Jpeg images stored on the file systems. It accepts external sharding information to load a singe shard. only
-/// \param rocal_context Rocal context
-/// \param source_path A NULL terminated char string pointing to the location on the disk
-/// \param json_path Path to the COCO Json File
-/// \param rocal_color_format The color format the images will be decoded to.
-/// \param shard_id Shard id for this loader
```

Comment:
Please check if Lines 210-215 needs to be removed

Reply:
Added it back. I think this wasnt supposed to be removed